cisco_duo/assets/logs/cisco-duo.yaml

```diff
@@ -0,0 +1,185 @@
+  type: double
+- groups:
+  - User
+  name: User Email
+  path: usr.email
+  source: log
+- groups:
+  - User
+  name: User Name
+  path: usr.name
+  source: log
+- groups:
+  - Event
+  name: Event Name
+  path: evt.name
+  source: log
+- groups:
+  - Event
+  name: Event Outcome
+  path: evt.outcome
+  source: log
+pipeline:
+  type: pipeline
+  name: Cisco Duo
+  enabled: true
+  filter:
+    query: "source:cisco-duo"
+  processors:
+    - type: date-remapper
+      name: Define `isotimestamp`, `ts` as the official date of the log
+      enabled: true
+      sources:
+        - isotimestamp
+        - ts
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: pipeline
+      name: Cisco Duo Authentication
+      enabled: true
+      filter:
+        query: "service:authentication"
+      processors:
+        - type: geo-ip-parser
+          name: GeoIP Parser for `access_device.ip`
+          enabled: true
+          sources:
+            - access_device.ip
+          target: access_device.geoip
+          ip_processing_behavior: do-nothing
+        - type: geo-ip-parser
+          name: GeoIP Parser for `auth_device.ip`
+          enabled: true
+          sources:
+            - auth_device.ip
+          target: auth_device.geoip
+          ip_processing_behavior: do-nothing
+        - type: attribute-remapper
+          name: Map `result` to `evt.outcome`
+          enabled: true
+          sources:
+            - result
+          sourceType: attribute
+          target: evt.outcome
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `email` to `usr.email`
+          enabled: true
+          sources:
+            - email
+          sourceType: attribute
+          target: usr.email
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `event_type` to `evt.name`
+          enabled: true
+          sources:
+            - event_type
+          sourceType: attribute
+          target: evt.name
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `user.name` to `usr.name`
+          enabled: true
+          sources:
+            - user.name
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Cisco Duo Activity
+      enabled: true
+      filter:
+        query: "service:activity"
+      processors:
+        - type: geo-ip-parser
+          name: GeoIP Parser for `access_device.ip.address`
+          enabled: true
+          sources:
+            - access_device.ip.address
```

Comment:
Replacing this would change the column value in the table that would not be clear to the user, hence I would recommend not doing this mapping.:
![Activity](https://github.com/DataDog/integrations-core/assets/35323912/4c418b78-e63c-4679-b976-7847dda9a9fd)